foundational_security/policy.hcl

```
@@ -429,17 +429,17 @@ policy "foundational_security" {
         description = "KMS controls"
         query "1" {
             description = "IAM customer managed policies should not allow decryption and re-encryption actions on all KMS keys"
-            query = "select 1;"
+            query = file("queries/kms/customer_policy_blocked_kms-actions.sql")
```

Nit:
```suggestion
            query = file("queries/kms/customer_policy_blocked_kms_actions.sql")
```
Rename the file as well

---

marked as resolved but I see it's still typoed

---

Sorry, didn't push yet just did it locally.